Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -79,6 +79,27 @@ open class ReadingThemesControlsViewController: UIViewController, AnalyticsConte
         NSObject.cancelPreviousPerformRequests(withTarget: self)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(_:)))
+        slider.addGestureRecognizer(tap)
+    }
+    
+    func handleTap(_ sender: UIGestureRecognizer) {
+        let pointTapped: CGPoint = sender.location(in: self.view)
```

Comment:
might want to do `location(in: slider)` since the tap gesture recognizer is added to the slider